src/helpers/colorHelpers.ts

```diff
@@ -0,0 +1,31 @@
+export const rainbowStop = (val:string) => {
```

Comment:
Pls add tests for ALL helpers you have added. It can be done w/ another PR, but create an issue for this.

Reply:
https://github.com/thorchain/asgardex-electron/issues/45